base/util.go

```diff
@@ -756,3 +758,9 @@ func GetExpvarAsString(mapName string, name string) string {
 		return ""
 	}
 }
+
+
+// TODO: temporary workaround until https://issues.couchbase.com/browse/MB-27026 is implemented
+func ExtractExpiryFromMemcachedRequest(rq *gomemcached.MCRequest) (expiry uint32) {
+	return binary.BigEndian.Uint32(rq.Extras[20:24])
```

Comment:
Maybe a safety check here that len(rq.Extras) >= 24?

Reply:
Fixed